packages/controller/test/lib/testConsole.js

```diff
@@ -639,6 +634,10 @@ function register(it, expect, context) {
         res = await cpPromise.exec(`"${process.execPath}" "${iobExecutable}" repo set stable`);
         expect(res.stderr).to.be.not.ok;
 
+        // remove local from active repos
+        res = await cpPromise.exec(`"${process.execPath}" "${iobExecutable}" repo unset local`);
+        expect(res.stderr).to.be.not.ok;
```

Comment:
The tests in general could check that at least system.config is changed correctly (not only here, also the other cases.Else we only know that action was not crashing